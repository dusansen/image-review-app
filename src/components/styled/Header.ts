import styled from 'styled-components';

export const Header = styled.header`
  text-align: left;
  width: 100%;
  color: #4a69bd;
`;
