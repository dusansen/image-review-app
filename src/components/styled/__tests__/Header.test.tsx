import renderer from 'react-test-renderer';
import { Header } from '../Header';

describe('Header', () => {
  it('should render Header with styles', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
