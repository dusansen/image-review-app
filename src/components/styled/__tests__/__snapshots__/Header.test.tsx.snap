// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Header with styles 1`] = `
<header
  className="sc-bdvvtL dkBQGu"
/>
`;
